linodecli/plugins/ssh.py

```diff
@@ -147,4 +152,7 @@ def parse_target_address(
         if ip.startswith("192.168"):
             continue
 
+        if getattr(parsed, "d"):
+            ip = ip.replace(".", "-") + ".ip.linodeusercontent.com"
```

Comment:
Would it make sense to fetch the RDNS the instance's public IPv4 address(es) using the `linode-cli linodes ips-list --json` command to avoid hardcoding it?

Reply:
Make sense